src/controller/ExampleOperationalCredentialsIssuer.cpp

```diff
@@ -263,6 +263,103 @@ CHIP_ERROR ExampleOperationalCredentialsIssuer::GenerateNOCChain(const ByteSpan
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR ExampleOperationalCredentialsIssuer::GenerateChipNOCChain(const ByteSpan & csrElements, const ByteSpan & csrNonce,
+                                                                     const ByteSpan & attestationSignature,
+                                                                     const ByteSpan & attestationChallenge, const ByteSpan & DAC,
+                                                                     const ByteSpan & PAI,
+                                                                     Callback::Callback<OnNOCChainGeneration> * onCompletion)
+{
+    VerifyOrReturnError(mInitialized, CHIP_ERROR_INCORRECT_STATE);
```

Comment:
Why are we copy/pasting all this code from GenerateNOCChain instead of at least using a helper function?  It's all the same code except in the end we're converting DER certs to Matter certs in this function, no?

Reply:
Fixed in https://github.com/project-chip/connectedhomeip/pull/20013